### PR TITLE
Basically, fix everything

### PR DESCRIPTION
**Pull request by tcitworld**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Documentation | Needs changes
| Translation   | no
| Fixed tickets | 
| License       | MIT

* Remove install.sh
* Assets are now provided
* Icons font issues are fixed

Should fix https://github.com/wallabag/wallabag/issues/2356, https://github.com/wallabag/wallabag/issues/2346, https://github.com/wallabag/wallabag/issues/2343

- [x] Fix documentation install instructions
- [x] Remove travis job (for now)

When updating source assets, you now also update generated ones.

### CODE REVIEW

**Comment by nerzhul** [comment]
Thanks this could be good because packaging is just a pain. I'm trying to port it on freebsd and use build methods in port it's just pure pain...
If a release build can be provided container all the required php deps we could provide a good quality port for FreeBSD.

Note: i'm nextcloud & rainloop maintainer on freebsd

File: app/Resources/static/themes/baggy/css/main.css [hunk 1]
```diff
@@ -838,7 +847,7 @@ a.add-to-wallabag-link-after::after {
 }
 
 .icon-twitter::before {
-  content: "\ea91";
+  content: "\ea96";
```
Comment (j0k3r) [inline app/Resources/static/themes/baggy/css/main.css]: 👍

**Comment by nicosomb** [comment]
👍

File: .travis.yml [hunk 2]
```diff
@@ -96,4 +96,4 @@ script:
     - if [[ $VALIDATE_TRANSLATION_FILE = '' || $ASSETS = '' ]]; then phpunit -v ; fi;
     - if [[ $CS_FIXER = run ]]; then php bin/php-cs-fixer fix src/ --verbose --dry-run ; fi;
     - if [[ $VALIDATE_TRANSLATION_FILE = run ]]; then php bin/console lint:yaml src/Wallabag/CoreBundle/Resources/translations -v ; fi;
-    - if [[ $ASSETS = build ]]; then node_modules/grunt/bin/grunt; fi;
+    - if [[ $ASSETS = build ]]; then grunt; fi;
```
Comment (tcitworld) [inline .travis.yml]: grunt (and everything else) is missing, we need to `npm i` before.

Comment (j0k3r) [inline .travis.yml]: On it.

**Comment by j0k3r** [comment]
On it.

**Comment by tcitworld** [comment]
- [ ] Use same icons on material than on baggy for public link/delete public link
- [ ] Fix shaarli/diaspora/carrot icons positions on material

File: .travis.yml [hunk 3]
```diff
@@ -95,4 +95,4 @@ script:
     - if [[ $VALIDATE_TRANSLATION_FILE = '' ]]; then phpunit -v ; fi;
     - if [[ $CS_FIXER = run ]]; then php bin/php-cs-fixer fix src/ --verbose --dry-run ; fi;
     - if [[ $VALIDATE_TRANSLATION_FILE = run ]]; then php bin/console lint:yaml src/Wallabag/CoreBundle/Resources/translations -v ; fi;
-    - if [[ $ASSETS = build ]]; then ./node_modules/grunt-cli/bin/grunt; fi;
+    - if [[ $ASSETS = build ]]; then ./node_modules/grunt-cli/bin/grunt tests; fi;
```
Comment (j0k3r) [inline .travis.yml]: No more build, just tests?

Comment (tcitworld) [inline .travis.yml]: Indeed, there could be errors of js not building too. I want to see how long this takes already.

**Comment by j0k3r** [comment]
:+1:

**Comment by tcitworld** [comment]
Now for some documentation fixing.

**Pull request merged**